packages/manager/src/utilities/testHelpers.tsx

```diff
@@ -152,6 +153,21 @@ export const renderWithTheme = (ui: any, options: Options = {}) => {
   return render(wrapWithTheme(ui, options));
 };
 
+/**
+ * Renders the given UI component within both the Formik and renderWithTheme.
+ *
+ * @param {React.ReactElement} ui - The React component that you want to render. This component
+ *                                  typically will be a part of or a whole form.
+ * @param {FormikConfig<T>} configObj - Formik configuration object which includes all the necessary
+ *                                      configurations for the Formik context such as initialValues,
+ *                                      validationSchema, and onSubmit.
+ */
+
+export const renderWithThemeAndFormik = <T extends FormikValues>(
+  ui: React.ReactElement,
+  configObj: FormikConfig<T>
+) => renderWithTheme(<Formik {...configObj}>{() => ui}</Formik>);
```

Comment:
`() => ui` can be simplified to just `ui`

```suggestion
) => renderWithTheme(<Formik {...configObj}>{ui}</Formik>);
```